charts/gxf/templates/configmap.yaml

```diff
@@ -9,3 +9,40 @@ data:
   {{ $name }}: |
     {{- (tpl $data $) | nindent 4 }}
   {{- end }}
+  logback-global.xml: |
+    <?xml version="1.0" encoding="UTF-8"?>
+      <configuration>
+
+        <appender name="CONSOLE" class="ch.qos.logback.core.ConsoleAppender">
+          <encoder>
+            <pattern>[%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n</pattern>
+          </encoder>
+        </appender>
+  {{- if .Values.logging.toFileEnabled }}
+        <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
+          <file>/var/log/osp/logs/{{ .Values.logging.filename | default .Release.Name }}.log</file>
+          <rollingPolicy class="ch.qos.logback.core.rolling.SizeAndTimeBasedRollingPolicy">
+            <fileNamePattern>/var/log/osp/logs/{{ .Values.logging.filename | default .Release.Name }}.log.%d{yyyy-MM-dd}.%i.log.gz</fileNamePattern>
+            <maxFileSize>100MB</maxFileSize>
+            <maxHistory>{{ .Values.logging.maxHistoryDays }}</maxHistory>
+            <totalSizeCap>{{ .Values.logging.totalSizeCap }}</totalSizeCap>
+          </rollingPolicy>
+          <encoder>
+            <pattern>[%d{yyyy-MM-dd HH:mm:ss.SSS}] [${HOSTNAME}] [%thread] %level %logger{36}@%method:%line - %msg%n</pattern>
+          </encoder>
+        </appender>
+  {{- end }}
+
+        <logger name="org.springframework" level="ERROR" />
+        <logger name="org.apache" level="ERROR" />
+        <logger name="com.zaxxer.hikari" level="ERROR" />
+        <logger name="org.quartz" level="ERROR" />
```

Comment:
Would be **very nice** to generate this based on a map in the values.yaml. But that is not necessarily part of this PR.